tests/providers/sftp/hooks/test_sftp.py

```diff
@@ -385,41 +385,45 @@ def test_invalid_ssh_hook(self, mock_get_connection):
         with pytest.raises(AirflowException, match="ssh_hook must be an instance of SSHHook"):
             connection = Connection(conn_id="sftp_default", login="root", host="localhost")
             mock_get_connection.return_value = connection
-            SFTPHook(ssh_hook="invalid_hook")  # type: ignore
+            with pytest.warns(DeprecationWarning, match=r"Parameter `ssh_hook` is deprecated.*"):
+                SFTPHook(ssh_hook="invalid_hook")
 
     @mock.patch("airflow.providers.ssh.hooks.ssh.SSHHook.get_connection")
     def test_valid_ssh_hook(self, mock_get_connection):
         connection = Connection(conn_id="sftp_test", login="root", host="localhost")
         mock_get_connection.return_value = connection
-        hook = SFTPHook(ssh_hook=SSHHook(ssh_conn_id="sftp_test"))
+        with pytest.warns(DeprecationWarning, match=r"Parameter `ssh_hook` is deprecated.*"):
+            hook = SFTPHook(ssh_hook=SSHHook(ssh_conn_id="sftp_test"))
         assert hook.ssh_conn_id == "sftp_test"
         assert isinstance(hook.get_conn(), paramiko.SFTPClient)
 
     def test_get_suffix_pattern_match(self):
         output = self.hook.get_file_by_pattern(TMP_PATH, "*.txt")
-        self.assertTrue(output, TMP_FILE_FOR_TESTS)
+        # In CI files might have different name, so we check that file found rather than actual name
+        assert output, TMP_FILE_FOR_TESTS
```

Comment:
Just for now I keep same logic for this test: "Returns any file without check expected".
Seems like in CI and local breeze there are different set of files as result

```python
assert output == TMP_FILE_FOR_TESTS
```
Pass in the local environment and failed in the CI with error

```text
E       AssertionError: assert 'remote_path.txt' == 'test_file.txt'
E         - test_file.txt
E         + remote_path.txt
```